hackernews-react-apollo/src/components/LinkList.js

```diff
@@ -1,26 +1,28 @@
 import { useQuery, gql } from "@apollo/client";
 // import React from 'react';
-import Link from "./link";
+// import Link from "./link";
 
-const FEED_QUERY = gql`
+const GET_USER = gql`
   {
-    books {
-      title
-      author
+    user(id: "2") {
```

Comment:
don't hardcode id
see how can you write queries with dynamic parameters.
It is straight forward